GiniVision/Classes/CameraViewController.swift

```diff
@@ -103,23 +120,36 @@ public typealias CameraErrorBlock = (_ error: CameraError) -> ()
     }
     
     // Output
-    fileprivate var successBlock: CameraSuccessBlock?
-    fileprivate var errorBlock: CameraErrorBlock?
+    fileprivate var successBlock: CameraScreenSuccessBlock?
+    fileprivate var failureBlock: CameraScreenFailureBlock?
     
     /**
-     Designated intitializer for the `CameraViewController` which allows to set a success block and an error block which will be executed accordingly.
+     Designated initializer for the `CameraViewController` which allows to set a success block and an error block which will be executed accordingly.
      
-     - parameter success: Success block to be executed when image was taken.
+     - parameter success: Success block to be executed when document was picked or image was taken.
      - parameter failure: Error block to be executed if an error occurred.
      
-     - returns: A view controller instance allowing the user to take a picture.
+     - returns: A view controller instance allowing the user to take a picture or pick a document.
      */
-    public init(success: @escaping CameraSuccessBlock, failure: @escaping CameraErrorBlock) {
+    public init(successBlock: @escaping CameraScreenSuccessBlock, failureBlock: @escaping CameraScreenFailureBlock) {
```

Comment:
This is the longest method ever... can you extract some functionality away from it?

Reply:
It is not possible as I commented above.